src/main/daml/Daml/Finance/Interface/Types/Date/FixingDates.daml

```diff
@@ -61,11 +59,13 @@ data FixingDates = FixingDates
       -- ^ The identifiers of the holiday calendars to be used for date adjustment (if any).
   deriving (Eq, Show)
 
--- | A day type classification used in counting the number of days between two dates.
-data DayTypeEnum
-  = Business
-    -- ^ When calculating the number of days between two dates the count includes only business
-    --   days.
-  | Calendar
-    -- ^ When calculating the number of days between two dates the count includes all calendar days.
+-- | Specifies the data required for a floating rate coupon.
+data FloatingRate = FloatingRate
+  with
+    referenceRateId : Text
+      -- ^ The identifier of the reference rate to be used for the coupon, e.g. Libor-3M.
+    referenceRateType : ReferenceRateTypeEnum
```

Comment:
it seems that we are injecting information on rates in a `Date` package, probably for lack of a better place. Do we have an alternative that keeps the `Date` package include only date-relevant types?

Reply:
It depends, does it need to be in `Interface.Types`? Currently, those are split by sub folder, i.e. one package for  `Interface.Types.Common` and another one for  `Interface.Types.Date`. That would mean a new package.

Alternatively, one could put them together with the util functions, but that would mean importing an implementation package every time you need these types.

Reply:
New interface package seems the way to go 😓

Reply:
Maybe a new one that can be used for market data for instrument modelling?

Reply:
We probably want to avoid a new package for every new `Types` sub-folder. Maybe this could also go as a module in the `Common` sub-folder?

Reply:
Alternatively, I could merge it as-is and open a follow-up Issue to deal with the general structure of our Types (and possibly move these ones according to what comes out there).

Reply:
We are basically looking for a place to store data types that are common across instrument definitions. What about a new package `Daml.Finance.Interface.Instrument.Types`?

I have the feeling that the new upgrading approach will make it easier for us to deal with this: we would lump all data types into the same `DAR` and reference it from the template and interface package